connection/database.feature

```diff
@@ -133,10 +133,10 @@ Feature: Connection Database
     Then connection does not have database: grakn
     Then session, open transaction of type; throws exception: write
 
-  @ignore-grakn-core
+  @ignore
```

Comment:
This test is failing in java because it doesn't throw the correct exception message (though it does throw). We need to fix this later.